### DISCUSSION TITLE
Welcome to phone-formatter-eth Discussions!

### DISCUSSION THREAD

**Discussion by HuluWZ** [HuluWZ/phone-formatter-eth#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HuluWZ** [comment]
Hi everyone! I'm excited to be part of this community. I'm currently working on a project involving phone number formatting, specifically for Ethiopian numbers, and I'm using an npm package designed for this purpose. I’d love to hear from others who have experience with phone number formatting or related tasks. How have you utilized phone number formatting in your projects?

- Do you have any tips or challenges you've faced while working with Ethiopian phone numbers or similar formats?

Looking forward to learning and sharing ideas with you all!

**Reply by eyopa21** [reply, marked as answer]
### Phone Number Type Detection
Differentiate between landline and mobile numbers based on prefixes.

### Number Formatting Options
Allow users to format numbers in different ways (e.g., `+251 912 345 678`, `0912-345-678`, etc.).

### Batch Processing
Enable formatting of multiple numbers at once.
